sweetalert2.d.ts

```diff
@@ -227,7 +227,13 @@ declare module 'sweetalert2' {
         'text' | 'email' | 'password' | 'number' | 'tel' | 'range' | 'textarea' | 'select' | 'radio' | 'checkbox' |
         'file' | 'url' | undefined;
 
-    export type SweetAlertDismissalReason = 'cancel' | 'close' | 'overlay' | 'esc' | 'timer';
```

Comment:
⚠️ (Small) Breaking change here. As it is exported, if an user has imported it direclty, it will break.

However, I'm not against its immediate deletion:

 - uses cases for importing it are frankly rare, so there are chances that nobody has already done that
 - keeping it for now in union with `SweetAlertDismissReason` will probably reduce the quality of autocomplete, as IDEs have a tendancy to give better results and "symbol inference" on simple types.

Reply:
Yup, I was thinking the same way. This is front-end, we need to break at least something :smiling_imp: